test/BUILD.gn

```diff
@@ -331,6 +328,10 @@ test("brave_unit_tests") {
     deps += [ "//brave/browser/playlist/test:unit_tests" ]
   }
 
+  if (enable_brave_referrals) {
+    deps += [ "//brave/components/brave_referrals/browser" ]
```

Comment:
`if (enable_brave_referrals) {` already exists below in `brave_unit_tests` target. please move this dep into it.